package/MDAnalysis/coordinates/chain.py

```diff
@@ -121,23 +121,19 @@ def filter_times(times, dt):
 
     # more then 2 unique time entries
 
-    # special case first times are equal
-    if not np.allclose(times[0][0], times[1][0]):
-        used_idx = [0, ]
-        offset = 1
-    else:
-        used_idx = [1, ]
-        offset = 2
-
-    for i, (first, middle, last) in enumerate(zip(times[:-2], times[1:-1], times[2:])):
-        if not np.allclose(middle[1] - middle[0], dt):
+    used_idx = [0, ]
```

Comment:
Is this really what we want? This means the first trajectory is always taken. Is this also OK in your use-case where the first part might fail and the simulation starts again from the initial structure?

Reply:
If the next part starts at the same time as the first one, it will replace it so that the first part gets excluded. This should cover the same function as the special case you had before.

Reply:
That is right. Thanks for the fix!